### PR TITLE
Rewriten shell scripts in python to make them compatible with windows

### PR DESCRIPTION
**Pull request by yanchenko-igor**
Created new script features/callback.py which replaces features/callback.sh

### CODE REVIEW

File: features/backup_create.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--datadir", required=True)
+    parser.add_argument("--dbname", required=True)
+    parser.add_argument("--walmethod", required=True, choices=("fetch", "stream", "none"))
+    args, _ = parser.parse_known_args()
+
+    command = ["pg_basebackup", "-D", args.datadir] + \
+              (["-X", args.walmethod] if args.walmethod != "none" else []) + \
+              ["-c", "fast", "-d", args.dbname]
+    subprocess.call(command)
```
Comment (CyberDem0n) [inline features/backup_create.py]: We don't want to lose the exit code.

File: features/environment.py [hunk 2]
```diff
@@ -623,7 +623,7 @@ def bootstrap_from_backup(self, name, cluster_name):
             'bootstrap': {
                 'method': 'backup_restore',
                 'backup_restore': {
-                    'command': 'features/backup_restore.sh --sourcedir=' + os.path.join(self.patroni_path,
+                    'command': 'features/backup_restore.py --sourcedir=' + os.path.join(self.patroni_path,
```
Comment (CyberDem0n) [inline features/environment.py]:
```python
'command': sys.executable + 'features/backup_restore.py...
```

The same applies to other places where we execute py files.

**Comment by CyberDem0n** [comment]
:+1:

**Comment by FxKu** [comment]
:+1:

**Comment by FxKu** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**